### ISSUE TITLE
[bitnami/clickhouse] When deploying clickhouse with multi shards and replicas with Zookeeper, I start getting warnings and errors

### ISSUE THREAD

**Issue by Kareem-Afaneh** [bitnami/charts#26444]
### Name and Version

bitnami/clickhouse-6.1.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1- Deploying the helm chart on EKS cluster, using this 

`oci://registry-1.docker.io/bitnamicharts/clickhouse`.

2- Use values file with these values: 

    resources:
      requests:
        cpu: "500m"
        memory: 1Gi
      limits:
        cpu: 1
        memory: 2Gi
    

- Use Ingress.

- Use Zookeeper

Then After start using the clickhouse, i start getting errors and warnings.

For example the logs of pod clickhouse-shard1-1 gives something like this: 
`<Warning> DNSResolver: Cannot resolve host (clickhouse-shard0-1.***.), error 0: Host not found.`  

Also, when i create DB for example it will be created on one of the replicas, so when I do a select from that DB, it only will work if the request hit the replica that the DB was created on. So the replicas are not synced.

### What do you see instead?

The pods are running fine, but facing warnings and errors when using clickhouse.

### Additional information

What I am doing wrong here? Is it about using the zookeeper? 

Basically, i want to sync all replicas so that when hitting a query, the result should be always the same, not based on which replica was chosen. In other words I want to do Horizontal Scaling.

Thanks!

**Comment by javsalgar** [comment]
Hi,

About the "host not found" issue it could clearly be a transient error, as kube-dns may not have the hostname available yet. About the database creation, did you check the database creation commands? In order for a database to be cluster-scoped, the regular CREATE DATABASE command was not enough.

https://clickhouse.com/docs/en/sql-reference/statements/create/database

**Comment by Kareem-Afaneh** [comment]
@javsalgar 

Yes, I noticed that the regular commands won't work with replicas and shards. 

Thanks for your response.

PS: I am closing this issue.
